open-metadata-implementation/server-chassis/server-chassis-spring/pom.xml

```diff
@@ -292,6 +292,11 @@
             <artifactId>slf4j-api</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
```

Comment:
we'll end up pulling in so much here with lucene - which already has some security exposures - and end up infecting all our components?

Reply:
This dependency should be removed